notebooks/113-image-classification-quantization/113-image-classification-quantization.ipynb

```diff
@@ -93,7 +93,7 @@
         "torch.onnx.export(model, dummy_input, onnx_model_path, verbose=True)\n",
         "\n",
         "# Run Model Optimizer to convert ONNX to OpenVINO IR.\n",
-        "!mo --framework=onnx --data_type=FP16 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+        "!mo --data_type=FP16 -m $onnx_model_path  --output_dir $MODEL_DIR"
```

Comment:
We have renamed `--data_type` to `--compress_to_fp16` in the recent version of master.

Reply:
Please do not mixup branches! These changes will be later, when we can test them on ci, for now it should save compatibility with 2022.2 as stable release